pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -22,51 +28,58 @@ import (
 	"k8s.io/kubernetes/pkg/kubectl/scheme"
 	"k8s.io/kubernetes/pkg/kubectl/util/templates"
 
-	v1 "github.com/openshift/client-go/image/clientset/versioned/typed/image/v1"
+	imageclient "github.com/openshift/client-go/image/clientset/versioned/typed/image/v1"
```

Comment:
Nit, the convention is for this to be named `imagev1client`.

Reply:
Done